lua_symbols.ldsym

```diff
@@ -0,0 +1,3 @@
+{
+	lua_*;
```

Comment:
might be best to scope this more for the long term if possible (I have no idea how many that would require)